src/wweb/onMessage/commands.ts

```diff
@@ -0,0 +1,97 @@
+import logsMessage from './debug/logs';
+import gptMessage from './gpt/gptMessage';
+import resumeMessage from './gpt/resumeMessage';
+import resumeStartMessage from './gpt/resumeStartMessage';
+import resumeStopMessage from './gpt/resumeStopMessage';
+import helpMessage from './helpMessage';
+import authorMessage from './meme/authorMessage';
+import drakeMessage from './meme/drakeMessage';
+import incrivelMessage from './meme/incrivelMessage';
+import quoteMessage from './meme/quoteMessage';
+import tigerMessage from './meme/tigerMessage';
+import tradeOfferMessage from './meme/tradeOfferMessage';
+
+const COMMANDS = {
```

Comment:
it can be an array as well, but in that way, we can call it easier in the future

Reply:
maybe it should be a list + it should bring the data from the command